quipucords/tests/api/details_report/test_details_report.py

```diff
@@ -89,6 +92,6 @@ def test_retrieve_csv(self, sources, django_client):
             "batata,tomate\r\n"
             "\r\n\r\n"
         )
-        assert response.text == expected_csv
+        assert response.content.decode() == expected_csv
```

Comment:
`.text` is yet another elegant property from requests we are losing

(*urge to hack django client response intensifies*)